packages/odyssey-design-tokens/src/space.json

```diff
@@ -1,9 +1,18 @@
 {
   "space": {
-    "m": "",
-    "l": "",
-    "xl": "",
-    "s": "",
-    "xs": ""
+    "fixed" : {
```

Comment:
@arnoldsandoval-okta thoughts here? See resultant use in the module.

Reply:
I'm not sure I would classify `rem` as fixed spacing because it can change based on the document font size yea? I immediately think of px when I see this. Perhaps call this `rootRelative` and below `parentRelative` to be clear